HelpSource/Classes/Pbind.schelp

```diff
@@ -89,6 +89,42 @@ Pbind(*[
 )
 ::
 
+
+subsection::Multiple Instruments
+
+Pbind cannot be used to play multiple instruments. The following will not work.
+
+code::
+Pbind(
+     \instrument, [\beep, \boop], // this line is wrong
+     \dur, 1,
+     \freq, 220
+ )
+::
+
+This is a limit of the implmentation, but also encourages the use of smaller more managable patterns.
+
+To help solve this the following function is provided, but it is just one way to solve the problem - feel free to copy it into your projects as needed!
```

Comment:
This si a little wordy for a helpfile. Maybe: "the following is one of many ways to handle multiple instruments in parallel"

Also, you can't just copy this into projects, because the environment variables may not work, eg. when you are inside patterns.